site/_quarto.yml

```diff
@@ -161,15 +161,17 @@ website:
         - guide/guides.qmd
         - text: "---"
         - text: "CONFIGURATION"
-        - guide/configuration/set-up-your-organization.qmd
+        - file: guide/configuration/manage-your-organization.qmd
```

Comment:
Given that this topic has child topics under it, the title should be "Managing your organization" and the file name should match that. 

(FWIW, "Onboarding users" likely should also be "Managing users" as we also cover removing users.)

Reply:
I will change both of those!